README.md

```diff
@@ -1,27 +1,39 @@
 # @most/adapter
 
-Mostjs emphasizes declarative event streams, which have many advantages, such as helping to avoid race conditions.  However, getting data into a mostjs event stream graph from 3rd party libraries that weren't designed with that approach in mind can be tricky.  Often, trying to force a library into that model leads to a messy solution.
+Most.js emphasizes declarative event streams, which have many advantages such as helping to avoid race conditions. However, getting data into a Most.js event stream graph from third-party libraries designed without that approach in mind can be tricky. Often, forcing a library into that model leads to a messy solution.
 
-Most-adapter provides a disciplined imperative API for getting external data into a mostjs stream, with the goal of making it simpler to integrate libraries that don't lend themselves to a declarative approach.
+@most/adapter provides a disciplined, imperative API for getting external data into a most.js stream. The goal is to make it simpler to integrate libraries that don’t lend themselves to a declarative approach. @most/adapter works well for cases where you can’t just implement the standard `run()` interface.
```

Comment:
The last sentence here is awesome, and _really_ helpful, imho: it represents one of the important criteria people should use in deciding to use `@most/adapter` or not.  I'm thinking we may want to make it even more prominent.

What do you think about adding a "When should I use it?" heading, and moving the last sentence above into it?

Reply:
@briancavalier  Yes, let me update the branch.

Reply:
Done!